src/leader.c

```diff
@@ -364,10 +364,15 @@ static void leaderExecV2(struct exec *req, enum pool_half half)
 	unsigned i;
 	int rv;
 
+#ifdef DQLITE_NEXT
```

Comment:
Not insisting as it is a temporary solution, just wanted to say that there is a way to orgranise it in a bit more different way:
```
#ifdef DQLITE_NEXT
leaderExecV2(req, POOL_TOP_HALF);
leaderExecV2(req, POOL_TOP_BOTTOM);
#else
queue_pool_work(...)
#endif 
```